maestral_qt/utils.py

```diff
@@ -161,7 +161,7 @@ def center_window(widget):
     x = (geometry.width() - widget.width()) / 2
     y = (geometry.height() - widget.height()) / 3
 
-    widget.move(x, y)
+    widget.move(int(x), int(y))
```

Comment:
Instead of converting to `int` by passing the floating point value, you can directly use integer division `//` in the preceding calculation.